### DISCUSSION TITLE
Tavsiye

### DISCUSSION THREAD

**Discussion by Ecenurcelik** [IBAX-io/go-ibax#1718] [Bounty Hunter]
Twetter Telegram ve  YouTube adresleriniz de Türkçe dil desteğini geliştirir ve hesaplarınızı Türkçe olarak açarsanız sizi platform olarak kimse geçemez.  Size olan inanç ve güvenimiz sonsuz . Başarılar dilerim ..

**Comment by Ecenurcelik** [comment]
Doğru söze ne hacet

**Reply by muhmmedcelik** [reply]
Soyadımiz aynı .
